src/TrackingCollectionTests/TrackingCollectionTests.cs

```diff
@@ -333,7 +333,7 @@ public void FilterTitleRun2()
         var total = 1000;
 
         var list1 = new List<Thing>(Enumerable.Range(1, total).Select(i => GetThing(i, i, i, "Run 1")).ToList());
-        var list2 = new List<Thing>(Enumerable.Range(1, total).Select(i => GetThing(i, i, i, "Run 2")).ToList());
+        var list2 = new List<Thing>(Enumerable.Range(1, total).Select(i => GetThing(i, i, i + 1, "Run 2")).ToList());
```

Comment:
So in this unit test, we are expecting the collection to reflect a list or Things with the Title "Run 2".
But if the updated times are the same, that should not be the case, so I am making a change to ensure the "Run 2" things have a newer `UpdatedAt` and now the unit test passes.